core-build/gulp-core-build/src/tasks/jest.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "Jest Task configuration",
+  "description": "Defines Jest task configuration",
+
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "ci": {
+      "description": "Indicate whether the test is running in a CI environment",
+      "type": "boolean"
+    },
+    "configFilePath": {
+      "description": "The path to a jest config file",
+      "type": "string"
+    },
+    "coverage": {
+      "description": "Indicates that test coverage information should be collected and reported in the output",
+      "type": "boolean"
+    },
+    "maxWorkers": {
+      "description": "Specifies the maximum number of workers the worker-pool will spawn for running tests.",
+      "type": "number"
+    },
+    "rootDir": {
+      "description": "The root directory for the project",
+      "type": "string"
+    },
+    "runInBand": {
+      "description": "Indicates that test coverage information should be collected and reported in the output",
```

Comment:
>Indicates that test coverage information should be collected and reported in the output [](start = 22, length = 87)

Fix this

Reply:
I'm not sure runInBand should be an option unless there is an absolute need. I think in rush scenarios it's a requirement.